app/src/main/java/com/github/sdp/ratemyepfl/database/ItemsRepositoryInterface.kt

```diff
@@ -3,9 +3,16 @@ package com.github.sdp.ratemyepfl.database
 import com.github.sdp.ratemyepfl.model.items.Classroom
 import com.github.sdp.ratemyepfl.model.items.Course
 import com.github.sdp.ratemyepfl.model.items.Restaurant
+import com.github.sdp.ratemyepfl.model.items.Reviewable
+import com.github.sdp.ratemyepfl.model.review.ReviewRating
 
 interface ItemsRepositoryInterface {
```

Comment:
I'll add my comment here so that everyone can see where is my point. We should manage here (i.e., at load time) the nullable values. I think that we don't need to keep null values in data. I think about a refactor for the repo to remove the null values as much as possible. Here, we can apply a filter to remove null values and return only non-null values.

Reply:
I agree that we could return lists of non-nullable items.